test/resources/accounts-strict-origin-off/alice/profile/card$.ttl

```diff
@@ -1,12 +1,11 @@
 @prefix : <#>.
 @prefix acl: <http://www.w3.org/ns/auth/acl#>.
-@prefix c: <https://localhost:7010/profile/card#>.
-@prefix c0: <https://localhost:8443/profile/card#>.
+@prefix c: <https://localhost:8443/profile/card#>.
 
-c:me
+:me
     acl:trustedApp
     [ acl:mode acl:Read, acl:Write; acl:origin <https://app.example.com> ].
-c0:me
+c:me
```

Comment:
Two different `me`s probably means trouble. Should be both the same, so both `:` / `<#>`.